apps/omg_watcher_info/lib/omg_watcher_info/db/eth_event.ex

```diff
@@ -195,6 +197,7 @@ defmodule OMG.WatcherInfo.DB.EthEvent do
           root_chain_txhash_event: root_chain_txhash_event,
           log_index: log_index,
           root_chain_txhash: root_chain_txhash,
+          eth_height: eth_height,
```

Comment:
this is pretty uncool ` get(root_chain_txhash_event) ||`

Reply:
the correct way of doing this would be to write a case statement
ethevent = case get(root_chain_txhash_event) do
nil -> create the strcut
event -> event
end

it's because of: https://erlef.github.io/security-wg/secure_coding_and_deployment_hardening/elixir_truthy